fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalBlockTags.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.convention.v2;
+
+import net.minecraft.block.AbstractBlock;
+import net.minecraft.block.Block;
+import net.minecraft.registry.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.convention.v2.TagRegistration;
+
+/**
+ * See {@link net.minecraft.registry.tag.BlockTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public final class ConventionalBlockTags {
+	private ConventionalBlockTags() {
+	}
+
+	// Natural stone-like blocks that can spawn in-world such as Stone, Granite, Diorite, and others.
+	// Not variations like Stone Bricks or Diorite Slabs.
+	public static final TagKey<Block> STONES = register("stones");
+
+	// Ores and ingots - broad categories
```

Comment:
This needs some clarification - is the tag of ore and ingot blocks? If it has nothing to do with ingots, they shouldn't be mentioned here.

Reply:
That's actually what Fabric API has right now as a comment
https://github.com/FabricMC/fabric/blob/3ccb77f2a4dd0bde504de1ca0ae4862afb616870/fabric-convention-tags-v1/src/main/java/net/fabricmc/fabric/api/tag/convention/v1/ConventionalBlockTags.java#L35

They must had copied and pasted from the item tag. I'll fix that now